components/suggestion/search_channel_suggestion/index.ts

```diff
@@ -5,10 +5,17 @@ import {connect} from 'react-redux';
 import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
 
 import {getDirectTeammate} from 'utils/utils';
+import {GlobalState} from 'types/store';
 
 import SearchChannelSuggestion from './search_channel_suggestion';
 
-const mapStateToProps = (state, ownProps) => {
+type OwnProps = {
+    item: {
```

Comment:
I think `item` is of type `Channel` here as well.

Reply:
Hi @devinbinnie  I already add this.